### ISSUE TITLE
butler-reverse inconsistent with documentation

### ISSUE THREAD

**Issue by bestlem** [alphapapa/bufler.el#91]
The description for customising  butler-reverse is
`Reverse group order after grouping buffers.`

But it dopes not reverse the groups - it sorts them by name

The code in bufler.el is
```
(when bufler-reverse
          (setf groups (nreverse (-sort #'format< groups))))
```

**Comment by alphapapa** [comment]
Yes, you seem to be right.  This is a feature due to be rewritten since having written `taxy`, which will replace and upgrade the grouping logic.  I intend to get around to that someday.

By the way, please note that the package is named `bufler` not `butler`.  As it says, "A butler for your buffers."
